asyncapi/quickbooks/Module.md

```diff
@@ -0,0 +1,64 @@
+## Overview
+The QuickBooks trigger allows you to listen to [QuickBooks webhook notifications](https://developer.intuit.com/app/developer/qbo/docs/develop/webhooks).
+
+After you subscribe for webhooks to receive event triggered callbacks for entities that your app needs to stay on top of. QuickBooks webhooks automatically notify you whenever data changes in your end-user’s QuickBooks Online company files.
+
+## Prerequisites
+Before using this trigger in your Ballerina application, complete the following:
+
+* Create a [QuickBooks](https://quickbooks.intuit.com/) account
+* Add listener endpoint URL  to QuickBooks webhooks.
+* [Optional] Select and Save Entities and their events.
+
+
+## Quickstart
+To use the QuickBooks trigger in your Ballerina application, update the .bal file as follows:
+
+### Step 1: Import the trigger
+First, import `ballerinax/trigger.quickBooks` module into the Ballerina project as follows.
+```ballerina
+    import ballerinax/trigger.quickBooks;
+```
+
+### Step 2: Create a new trigger instance
+Initialize the trigger by providing the QuickBooks company ids in the listener config and the port number where your trigger will be running. You can also pass a http:Listener instance instead of the port number with the listener config. The QuickBooks companyId is shown in your app with the company list.
```

Comment:
Replace "a http:Listener" with "an http:Listerner".

When it is an abbreviation, whether to use "a" or "an" depends on how it sounds when you read out the letters rather than whether the first letter is a vowel or a consonant.